### DISCUSSION TITLE
Enabling commenting actions in `.vimrc` and `.ideavimrc`

### DISCUSSION THREAD

**Discussion by jphalip** [JetBrains/ideavim#1044] [Q&A]
If you edit `.vimrc` or `.ideavimrc` in IntelliJ then you get some nice syntax highlighting, which means that IntelliJ (or perhaps IdeaVim) recognizes the file type. However, if you try to comment/uncomment code by pressing `CMD-/` (on Mac OS) or `gc` (using the `commentary` extension), then nothing happens.

Is there a way to enable commenting in those files, or is that perhaps a missing feature?

**Comment by Old-Farmer** [comment]
Maybe reason is that Intellij ides do not have the viml filetype ?
